### ISSUE TITLE
Localization

### ISSUE THREAD

**Issue by tomazfb** [vincentjames501/angular-cron-gen#20]
Hello,

I'm not sure it's an issue but I couldn't found localization/i18n support. 

It would be great if it have support for it or if we could at least set the day names and some constants in the cronOptions parameter.

**Comment by vincentjames501** [comment]
I agree, this would be nice to have. I may take a crack at implementing this but if you'd like to submit a PR in the meantime that'd be great!
